packages/editor-ui/src/views/SettingsUsersView.vue

```diff
@@ -52,7 +52,23 @@
 				@copyPasswordResetLink="onCopyPasswordResetLink"
 				@allowSSOManualLogin="onAllowSSOManualLogin"
 				@disallowSSOManualLogin="onDisallowSSOManualLogin"
-			/>
+			>
+				<template #actions="{ user }">
+					<n8n-select
+						:modelValue="user.globalRole.name"
+						@update:modelValue="($event: IRole) => onRoleChange(user, $event)"
+						:disabled="usersStore.isMember"
```

Comment:
I'd rather make this check inclusive rather than exclusive. We should check for `:disabled="!usersStore.isInstanceOwner"` instead to avoid issues.

Reply:
Later we need to change this for sure but for now I just wanted to exclude `member` because both `owner` and `admin` can change user roles

I'll introduce a new role check in user store for `admin` and update the condition

Reply:
Sorry, it's ok like this, so only members can NOT change roles.
In the future we need to linclude all other roles who can not change roles

Reply:
I understand that it works, just want to be sure that we don't introduce mistakes in the future. This would be a very good use case to use a scope to determine if you can change the role of a member.

Reply:
Good catch, updated to use scope permission